src/components/interfaces/MOBILE_API.xml

```diff
@@ -2941,6 +2976,9 @@
         </param>
         <param name="state" type="RadioState" mandatory="false">
         </param>
+        <param name="sisData" type="SisData" mandatory="false">
```

Comment:
Add `since="5.0"` to new parameters in existing structs/functions

Reply:
@jacobkeeler Fixed at commit fd9f349304e4d47d25d1fed79a0442a599717de8